src/demo/polymer-lint.ts

```diff
@@ -12,17 +12,14 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import {Analyzer, Document, FSUrlLoader, PackageUrlResolver, Severity, Warning, WarningPrinter} from '../index';
+import {Analyzer, Document, Severity, Warning, WarningPrinter} from '../index';
 
 /**
  * A basic demo of a linter CLI using the Analyzer API.
  */
 async function main() {
   const basedir = process.cwd();
-  const analyzer = new Analyzer({
-    urlLoader: new FSUrlLoader(basedir),
-    urlResolver: new PackageUrlResolver()
-  });
+  const analyzer = Analyzer.forDirectory(basedir);
```

Comment:
yea, again, this method name feels off :/

Reply:
Changed!